domains/measurements/Measurements.API/API/MeteringPoints/Api/v2024_01_10/Controllers/MeteringPointsController.cs

```diff
@@ -0,0 +1,51 @@
+using System.Linq;
+using System.Threading.Tasks;
+using API.MeteringPoints.Api.v2024_01_10.Dto.Responses;
+using API.MeteringPoints.Api.v2024_01_10.Dto.Responses.Enums;
+using Asp.Versioning;
+using EnergyOrigin.TokenValidation.Utilities;
+using EnergyOrigin.TokenValidation.Values;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.MeteringPoints.Api.v2024_01_10.Controllers;
+
+[Authorize]
+[ApiController]
+[ApiVersion("20240110")]
+[Route("api/meteringpoints")]
+public class MeteringPointsController : ControllerBase
+{
+    private readonly Meteringpoint.V1.Meteringpoint.MeteringpointClient _client;
+
+    public MeteringPointsController(Meteringpoint.V1.Meteringpoint.MeteringpointClient client)
+    {
+        _client = client;
+    }
+
+    /// <summary>
+    /// Get metering points from DataHub2.0
+    /// </summary>
+    /// <response code="200">Successful operation</response>
```

Comment:
Should it be documented that 200 also could be an empty list if no meteringpooint exists?

Reply:
Sure - you mean in the response summary there?